src/Microsoft.SqlTools.ServiceLayer/DacFx/ExtractOperation.cs

```diff
@@ -37,7 +37,7 @@ public override void Execute()
             }
             catch
             {
```

Comment:
Using Version.TryParse would be more concise and readable since it avoid the try{} catch{}.